src/main/java/hibernatehw/controllers/CarController.java

```diff
@@ -0,0 +1,43 @@
+package hibernatehw.controllers;
+
+import hibernatehw.dto.CarRequestDto;
+import hibernatehw.dto.CarResponseDto;
+import hibernatehw.mapper.CarMapper;
+import hibernatehw.models.Car;
+import hibernatehw.service.CarService;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.validation.Valid;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cars")
+public class CarController {
+    private final CarService carService;
+    private final CarMapper carMapper;
+
+    public CarController(CarService carService, CarMapper carMapper) {
+        this.carService = carService;
+        this.carMapper = carMapper;
+    }
+
+    @PostMapping("/add-car")
```

Comment:
No. See here https://restfulapi.net/resource-naming/ (**Never use CRUD function names in URIs**)
```suggestion
    @PostMapping("/")
```

Reply:
хорошо, передаелаю